fourkitchens/suhumsci
# Lando

# READY FOR REVIEW

## Summary
Wraps up the lando setup integration for SUhumsci

## Need Review By (Date)
TBD

## Urgency
medium

## Steps to Test
1. Go to the lando readme file in `/lando/README.md` and follow the setup instructions.
2. Confirm you are able to get a site fully up and running without error related to building site dependencies, building a lando instance with the appserver and database services running appropriately, and are able to sync the site from the remote with BLT.
3. Without rebuilding or shutting down lando, run `lando blt drupal:sync --site=[site name]` for a different site and confirm you are able to have both running at the same time.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
